src/agent/src/config.rs

```diff
@@ -84,6 +86,8 @@ pub struct AgentConfig {
     pub supports_seccomp: bool,
     pub container_policy_path: String,
     pub aa_kbc_params: String,
+    pub https_proxy: String,
+    pub no_proxy: String,
```

Comment:
I think these should be `Url`'s so we can use the `url` crate so we can call `url::Url::parse(value)?` on them.

Reply:
> I think these should be `Url`'s so we can use the `url` crate so we can call `url::Url::parse(value)?` on them.

Yes, it make sense to do more checks on these inputs.
`url::Url::parse()` will require a `scheme` like `http` seperated by `:`, it will be ok for `https_proxy`. we can add a `get_url_value()` based on current `get_string_value()` with url parse check.
But for `no_proxy` it will be a list of IP with CIDR/asterisks or domain names, `url::Url::parse()` can not check them correctly.